src/Components/Facility/FacilityConfigure.tsx

```diff
@@ -109,7 +109,7 @@ export const FacilityConfigure = (props: IProps) => {
     setIsLoading(false);
     if (res?.ok) {
       Notification.Success({
-        msg: "Facility updated successfully",
+        msg: "Facility middleware updated successfully",
```

Comment:
Lets move strings to `i18n`

Reply:
yeah sure

Reply:
@bodhish should we also keep translations in other languages  ?

Reply:
yes if possible, not mandatory.

Reply:
Okay